api-index-v1.json

```diff
@@ -30782,7 +30782,7 @@
       "options": {
         "csharp_namespace": {
           "valueCounts": {
-            "Google.Cloud.Compute.V1Small": 1
+            "Google.Cloud.Compute.V1small": 1
```

Comment:
I would expect this one will be autogenerated?

Reply:
hm... I don't even know how this got here, I did not touch this file... good catch.###